src/linpde_gp/randprocs/kernels/_independent_multi_output.py

```diff
@@ -0,0 +1,52 @@
+import functools
+from typing import Optional, Sequence
+
+import jax
+from jax import numpy as jnp
+import numpy as np
+
+from ._jax import JaxKernel
+
+class IndependentMultiOutputKernel(JaxKernel):
+    def __init__(self, kernels: Sequence[JaxKernel]):
```

Comment:
Can we pass kernels by varargs here?
```suggestion
    def __init__(self, *kernels: JaxKernel):
```